assets/show-more.js

```diff
@@ -0,0 +1,26 @@
+class ShowMoreButton extends HTMLElement {
+  constructor() {
+    super();
+    this.addEventListener('click', (event) => {
+      if (event.target.nodeName === 'SPAN' && event.target.closest('[id^="Show-More-"]')) {
```

Comment:
Do we need separate event handling for `span` and `button`? I think it would be better to just attach 1 event listener to the button element.

Reply:
Yes that is because when you use space/enter the node type is `button` and when you click the type is `span

 `console.log(event.target.nodeName, 'nodename')`

https://screenshot.click/25-01-ae1u9-y4ui3.png

Reply:
Ok in that case, we may want to more explicitly separate the different types of events (keyboard vs. click). Relying on `target.nodeName` to distinguish event types doesn't seem like the correct way of achieving this.
```
const button = this.querySelector('button');
button.addEventListener('click', event => {
  this.expandShowMore(button);
});
this.addEventListener('keydown', event => {
  if (event.code.toUpperCase() == 'SPACE' || event.code.toUpperCase() == 'ENTER') {
    this.expandShowMore(button);
    const nextElementToFocus = event.target.parentNode.parentNode.querySelector(".facets__item-show-more")
    if (!nextElementToFocus.classList.contains('.hidden')) {
      nextElementToFocus.querySelector('input').focus();
    }
  }
});
```

Reply:
Good catch! Before the event listener was on the form so I wanted to ensure we were on the correct element. I think now we can just use the click. We dont need to listen for keydown 🤔 Seems to work on tab with click

https://github.com/Shopify/dawn/pull/1443/commits/6e54b9142de6575847bc95e0d9239299387b9151